pages/about.html

```diff
@@ -1,12 +1,34 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+	<meta charset="UTF-8">
+	<link href="/styles/styles.css" rel="stylesheet">
+	<link href="/styles/about.css" rel="stylesheet">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>About</title>
 </head>
-<body>
-  
+
+<body class="container">
```

Comment:
Putting a class on `body` is a little redundant, since there is only one per page (unless your page is _super_ invalid) and you can just select by the element in your CSS.